dockstore-client/src/main/java/io/github/collaboratory/wdl/WDLClient.java

```diff
@@ -101,12 +101,12 @@ public void downloadFiles() {
     @Override
     public File provisionInputFiles() {
         // Get list of input files
-        Bridge bridge = new Bridge(localPrimaryDescriptorFile.getParent());
         Map<String, String> wdlInputs = null;
+        WdlBridge wdlBridge = new WdlBridge();
         try {
-            wdlInputs = bridge.getInputFiles(localPrimaryDescriptorFile);
-        } catch (NullPointerException e) {
-            exceptionMessage(e, "Could not get WDL imports: " + e.getMessage(), API_ERROR);
+            wdlInputs = wdlBridge.getInputFiles(localPrimaryDescriptorFile.getAbsolutePath());
+        } catch (Exception ex) {
+            exceptionMessage(ex, "Problem parsing WDL file: " + ex.getMessage(), API_ERROR);
```

Comment:
This exception is likely to stay in for a long time, let's get a bit more specific (multi-catch may help you)

Reply:
Oops forgot to change this